src/components/App/App.css

```diff
@@ -18,16 +18,15 @@
 
 .body {
   padding: 50px;
+  margin: 0 auto;
   flex-grow: 1;
+  width: 100%;
+
+  max-width: 1280px;
```

Comment:
This wont work and yes, it's not working. Can you use the previous method below to specify 1280px.